concurrent/actor/eventloop/EventLoop.java

```diff
@@ -32,9 +35,10 @@
 public class EventLoop {
     private static final Logger LOG = LoggerFactory.getLogger(EventLoop.class);
 
-    private final TransferQueue<Runnable> eventQueue = new LinkedTransferQueue<>();
-    private final LogicalTimerQueue<Runnable> timerQueue = new LogicalTimerQueue<>();
+    private final TransferQueue<Pair<Runnable, Consumer<Exception>>> events = new LinkedTransferQueue<>();
+    private final LogicalTimerQueue<Runnable> timers = new LogicalTimerQueue<>();
```

Comment:
`timers` need to be able to handle errors as well.

Reply:
I'm thinking of what should we do when a timer task throws an exception. Should we just trigger the `exception` method of the Actor the same way we do for normal message?

Reply:
I've agreed with @lriuui0x0 to take care of the issue on a separate PR.